lib/linguist/languages.yml

```
@@ -5330,6 +5330,39 @@ Python traceback:
   tm_scope: text.python.traceback
   ace_mode: text
   language_id: 304
+Pylar:
+  type: programming
+  color: #ff6633
+  extensions:
+  - ".pyl"
+  - ".pylar"
+  - ".teide"
+  filenames:
+  - Import-Libraries
+  - Loop-Label-end
+  - Loop-Label pass
+  - dijkstra
+  - for-end
+  - for-pass
+  - run-server
+  - while-end
+  - while-pass
+  interpreters:
+  - pylar
+  aliases:
+  - pylar
+  - pyl
+  - teide
+  - pyrus
+  extensions:
+    - ".pylar"
+    - ".pyl"
+    - ".teide"
+    - ".pyrus"
+  color: "#fed659"
+  ace_mode: text
+  tm_scope: source.pylar
+  language_id: 21
```

```suggestion
  language_id: 383531292
```
The ID should be a hash of the language name.

---

Thanks

---

Nope. Everything needs to be fixed in _this_ PR.